### ISSUE TITLE
SR not downloading some episodes and I dont know why

### ISSUE THREAD

**Issue by loungebob** [SiCKRAGETV/sickrage-issues#3289]
SR Version:  BRANCH: (master) / COMMIT: (30435190ef8880901df555bcda0d46902ea81fb5)  

Should download but doesnt automatically:
Blood & Oil s01e03
Marvel's Guardians of the Galaxy s01e04
``` 
Backlog forced says:
2015-10-14 13:16:53 INFO     SEARCHQUEUE-BACKLOG-293614 :: [KickAssTorrents] :: No needed episodes found during backlog search for: [Marvel's Guardians of the Galaxy]
2015-10-14 13:16:53 INFO     SEARCHQUEUE-BACKLOG-293614 :: [KickAssTorrents] :: Performing episode search for Marvel's Guardians of the Galaxy
2015-10-14 13:16:53 INFO     SEARCHQUEUE-BACKLOG-293614 :: Beginning backlog search for: [Marvel's Guardians of the Galaxy]
2015-10-14 13:11:08 INFO     SEARCHQUEUE-BACKLOG-295555 :: [KickAssTorrents] :: No needed episodes found during backlog search for: [Blood & Oil]
2015-10-14 13:11:08 INFO     SEARCHQUEUE-BACKLOG-295555 :: [KickAssTorrents] :: Ignoring Blood.and.Oil.S01E03.HDTV.XviD-FUM[ettv] because its not a valid scene release that we want, ignoring it
2015-10-14 13:11:08 INFO     SEARCHQUEUE-BACKLOG-295555 :: [KickAssTorrents] :: Quality of Blood.and.Oil.S01E03.HDTV.XviD-FUM[ettv] is SDTV
2015-10-14 13:11:08 INFO     SEARCHQUEUE-BACKLOG-295555 :: [KickAssTorrents] :: Ignoring result Blood.and.Oil.S01E03.720p.mHD.DailyFliX.XviD.avi because we don't want an episode that is Unknown
2015-10-14 13:11:08 INFO     SEARCHQUEUE-BACKLOG-295555 :: [KickAssTorrents] :: Ignoring result Blood.and.Oil.S01E03.720p.WEB-DL.x265-RMTeam.[720P][HEVC] because we don't want an episode that is 720p WEB-DL
2015-10-14 13:11:08 INFO     SEARCHQUEUE-BACKLOG-295555 :: [KickAssTorrents] :: Ignoring result Blood.and.Oil.S01E03.720p.WEB-DL.DD5.1.H.264-KiNGS[rartv] because we don't want an episode that is 720p WEB-DL
2015-10-14 13:11:08 INFO     SEARCHQUEUE-BACKLOG-295555 :: [KickAssTorrents] :: Ignoring result Blood.and.Oil.S01E03.1080p.WEB-DL.DD5.1.H.264-KiNGS[rartv] because we don't want an episode that is 1080p WEB-DL
2015-10-14 13:11:08 INFO     SEARCHQUEUE-BACKLOG-295555 :: [KickAssTorrents] :: Performing episode search for Blood & Oil
2015-10-14 13:11:08 INFO     SEARCHQUEUE-BACKLOG-295555 :: Beginning backlog search for: [Blood & Oil]
```
Even though those episodes are available:
Blood.and.Oil.2015.S01E03.HDTV.x264-KILLERS[ettv].mp4 
Guardians.of.the.Galaxy.S01E04.Take.the.Milano.and.Run.720p.WEB-DL.x264.AAC.mp4
Marvels.Guardians.of.the.Galaxy.S01E04.Take.the.Milano.and.Run.720p.WEB-DL.DD5.1.H.264-iT00NZ[rartv]

Any one know why this is?

**Comment by SiCKRAGEB0T** [comment]
## Thanks for the bug/issue report
Please make sure to submit all of the [required information](https://github.com/SiCKRAGETV/sickrage-issues/blob/master/README.md), or we won't be able to help you!
Provide as **much** information as you can such as debug logs, commit hash, and tell us how to replicate the problem.For basic **questions** use the [ [forum] ](https://sickrage.tv/forums/), and for **Feature Requests** [ [Feathub] ](http://feathub.com/SiCKRAGETV/SickRage)
Also make sure the title accurately describes your issue. ("SR not working" etc. is not ok!)

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist)

## Many issues can simply be solved by:
- Making sure you update to the latest version.
- Have you tried turning your device off and on again?
- Using the **search** function to see if this issue has already been reported/solved.
- Do a [GIT reset](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes#update-problems-try-this).
- Checking the [ [Wiki] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki) for [ [Guides] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Installation-&-Configuration-Guides), [ [FAQS] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes) [ [Main settings] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Settings-explained), [ [Show settings] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Show-settings-explained) [ [Remaining settings] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Remaining-settings-explained)
- Shutting down SickRage and removing the `cache/mako` & `cache/sessions` folders.
- If SickRage doesnt find any episodes you probably need to :
 - add a [Scene Exception](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Scene-exceptions-and-numbering)
 - Set the [Default Episode Status](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes#newly-aired-shows-are-not-downloading-and-set-to-skippedignored) to `WANTED`
 - Make sure you understand how [quality detection](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Quality-Settings#quality-detectiondetermination) works during searches.
 - Shutdown SickRage and remove the `cache.db` file.
- If you use a QNAP and have SSL errors try this [package](http://apps.qnap.community/11-community/17-qsickrage).
- If you have a `'ascii' codec can't encode character` error, than set your Locale to [UTF-8](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes#i-have-problems-with-special-characters-%C3%A9-etc-what-can-i-do)

*ps. If you enjoy Sickrage and like the support provided, than please consider making a [donation](https://github.com/SiCKRAGETV/SickRage/wiki/Donations) to keep SickRage alive.*

**Comment by neoatomic** [comment]
Make sure you add a scene exception and that the quality you have set matches with the downlaods that can be found on KAT. 
https://github.com/SiCKRAGETV/sickrage-issues/wiki/Scene-exceptions-and-numbering

**Comment by loungebob** [comment]
ok thanks, that worked. but really, shouldnt it know that marvels and marvel's and blood & oil and blood & oil 2015 is the same?

also, would you know the answer to this h$$ps://sickrage.tv/forums/forum/help-support/general-support/29199-get-sr-to-download-a-specific-web-dl-release ?

**Comment by duramato** [comment]
@loungebob set a required word for a the show you want

**Comment by loungebob** [comment]
@duramato you mean for this issue or for the one I posted the link to the sickrage forum?

**Comment by neoatomic** [comment]
@loungebob Well its impossible to know all the alternate names that up-loaders use or a show...

Below you find a list of all the scene exceptions, see if you can add it there so other users dont have that problem in the future. 
https://github.com/SiCKRAGETV/sb_tvdb_scene_exceptions/blob/gh-pages/exceptions.txt

**Comment by loungebob** [comment]
uh oh I dont know how and I dont want to mess the list up....

btw, I know but I was talking in this case specifically. any s-genitive is usually (for the sake of file naming and system limitations) just dropped (e.g. marvel's becomes marvels) and the ampersand should automatically be checked for the AND letters. 

anyway, now that I know what it was and how to fix it, all good.

**Comment by duramato** [comment]
@loungebob I'm saying for the one on the forum ,if you want to always download a certain release ,add a required word .Like if you always want to download FUM releases add FUM  to the required words...

**Comment by loungebob** [comment]
@duramato thanks, I know I can do that but it refutes the purpose of using something like SR. I collect way more than 50 shows and I dont want to edit each of them to fix the issue with low bitrate (and low quality web-dl). 
It would be much easier to include a mechanism to chose file quality further than just WEB/HDTV/BR....
like based on file size (higher bitrate and the same resolution with the same codec equals higher file size), or defining the min. bitrate (SR would download multiple web-dl versions and keeps only the one corresponding to that bitrate requirement)...maybe there's other ways to automate this.
Using the release group is unreliable since they tend to come and go, change names, split (especially in the web-dl area)

**Comment by fernandog** [comment]
@loungebob h265 has lower size that h264 and same quality. how to handle that?! we can't use size. also providers don't show bitrate. We need to make things uniform.

**Comment by loungebob** [comment]
@fernandog hence, my remark, using the same codec. but the same goes for h/x265, lower file size will have lower bitrate, higher file size will have higher bitrate. all one has to do is to differentiate between those two codecs. then, one can still say, download only the higher bitrate version of both, SR looks for both versions of the release and downloads the larger file size for each, x265 and x264.
there has to be some more automation here. if I have to do this for all the web-dl manually, the whole idea is kind of moot. btw, file sizes also work for BR and almost always for HDTV releases. 
I cant believe no one has had an issue with this. there's so much low file size, low quality releases out there lately, using the file size (or any other determination factor, there's probably a way) just makes sense to me. it's how I manually sort in KAT or Newsgroups.

**Comment by loungebob** [comment]
seriously. it's ridiculous I have to add a scene exception now for bobs burgers because suddenly SR cant find it while looking for bob's burgers....

**Comment by loungebob** [comment]
2015-10-19 17:17:40 INFO     SEARCHQUEUE-BACKLOG-267777 :: [KickAssTorrents] :: No needed episodes found during backlog search for: [Girl Meets World]
2015-10-19 17:17:40 INFO     SEARCHQUEUE-BACKLOG-267777 :: [KickAssTorrents] :: Ignoring result Girl.Meets.World.S02E21.Girl.Meets.Texas.Part.2.iTunes.1080p-DCMagic because we don't want an episode that is Unknown
2015-10-19 17:17:40 INFO     SEARCHQUEUE-BACKLOG-267777 :: [KickAssTorrents] :: Ignoring result Girl.Meets.World.S02E22.Girl.Meets.Texas.Part.3.720p.Webrip.AAC.X264 because we don't want an episode that is 720p WEB-DL
2015-10-19 17:17:40 INFO     SEARCHQUEUE-BACKLOG-267777 :: [KickAssTorrents] :: Ignoring result Girl.Meets.World.S02E21.Girl.Meets.Texas.Part.2.720p.Webrip.AAC.X264 because we don't want an episode that is 720p WEB-DL
2015-10-19 17:17:40 INFO     SEARCHQUEUE-BACKLOG-267777 :: [KickAssTorrents] :: Ignoring result Girl.Meets.World.S02E20.Girl.Meets.Texas.Part.1.720p.WEB-DL.DD5.1.H.264-TVSmash because we don't want an episode that is 720p WEB-DL
2015-10-19 17:17:40 INFO     SEARCHQUEUE-BACKLOG-267777 :: [KickAssTorrents] :: Ignoring result Girl.Meets.World.S02E20.Girl.Meets.Texas.Part.1.1080p.WEB-DL.DD5.1.x264-TVSmash because we don't want an episode that is 1080p WEB-DL
2015-10-19 17:17:39 INFO     SEARCHQUEUE-BACKLOG-267777 :: [KickAssTorrents] :: Performing episode search for Girl Meets World
2015-10-19 17:17:39 INFO     SEARCHQUEUE-BACKLOG-267777 :: Beginning backlog search for: [Girl Meets World]

the episodes are on KAT. I dont even know what scene exception to add so it would find them....

**Comment by neoatomic** [comment]
it's ridiculous that you ask a question that you answer yourself...
Might also try to remove the cache.db and cache folder.

**Comment by loungebob** [comment]
@neoatomic No it is not. It worked fine for 6 seasons and now it needs a scene exception. So something changed and not for the better, pointing this out is not at all ridiculous. Besides, I didnt ask a question in that post, I made a statement so someone else having the same issue could find it.

As for the real question in the 2nd post, this is not solved. I dont know what to add or change so SR will download Girl Meets World. Closing this topic is premature.
